modules/openapi-generator/src/main/resources/go-experimental/model.mustache

```diff
@@ -35,14 +52,14 @@ type {{classname}} struct {
 {{#description}}
 	// {{{description}}}
 {{/description}}
-	{{name}} *{{{dataType}}} `json:"{{baseName}},omitempty"{{#withXml}} xml:"{{baseName}}{{#isXmlAttribute}},attr{{/isXmlAttribute}}"{{/withXml}}{{#vendorExtensions.x-go-custom-tag}} {{{.}}}{{/vendorExtensions.x-go-custom-tag}}`
-{{#isNullable}}	isExplicitNull{{name}} bool `json:"-"{{#withXml}} xml:"-"{{/withXml}}`{{/isNullable}}
+	{{name}} {{^required}}*{{/required}}{{{dataType}}} `json:"{{baseName}}{{^required}},omitempty{{/required}}"{{#withXml}} xml:"{{baseName}}{{#isXmlAttribute}},attr{{/isXmlAttribute}}"{{/withXml}}{{#vendorExtensions.x-go-custom-tag}} {{{.}}}{{/vendorExtensions.x-go-custom-tag}}`
 {{/vars}}
 }
 {{/isEnum}}
 
 {{^isEnum}}
 {{#vars}}
+{{^required}}
```

Comment:
I'd prefer if we preserved at least `Get{{name}}` and `Set{{name}}` methods for required fields. It would make it possible to work with the generated library in a consistent way (using getters and setters for all fields) instead of using getters/setters for some fields and accessing other fields directly.

Reply:
Sounds reasonable, on it :-)

Reply:
Done.